docs/serving/using-custom-metrics-API.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Using the custom-metrics API"
+weight: 10
+type: "docs"
+---
+
+The `custom-metrics` API is installed by default when you install Knative Serving.
+
+>**IMPORTANT:** If you already have an existing `custom-metrics` API implementation in your cluster, you must delete this so that Knative Serving can use the default Knative Serving installation of the `custom-metrics` API. Enabling both implementations can cause issues.
```

Comment:
How is this scenario (having an existing API implementation) likely to come about? Might be nice to signal how users could know if this was likely to apply to them, if possible.